howto/setup-state-store/setup-postgresql.md

```diff
@@ -0,0 +1,56 @@
+# Setup PostgreSQL
+
+This article provides guidance on configuring a PostgreSQL state store.
+
+## Create a PostgreSQL Store
+Dapr can use any PostgreSQL instance. If you already have a running instance of PostgreSQL, move on to the [Create a Dapr component](#create-a-dapr-component) section.
+
+1. Run an instance of PostgreSQL. You can run a local instance of PostgreSQL in Docker CE with the following command:
+
+     This example does not describe a production configuration because it sets the password in plain text and the user name is left as the PostgreSQL default of "postgres".  
+
+     ```bash
+     docker run -p 5432:5432 -e POSTGRES_PASSWORD=example postgres
+     ```
+
+2. Create a database for state data.
+Either the default "postgres" database can be used, or create a new database for storing state data.
+
+    To create a new database in PostgreSQL, run the following SQL command:
+
+    ```SQL
+    create database dapr_test
+    ```
+
+## Create a Dapr component
+
+Create a file called `postgres.yaml`, and paste the following and replace the `<CONNECTION STRING>` value with your connection string. If you want to also configure PostgreSQL to store actors, add the `actorStateStore` configuration element shown below.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: statestore
+spec:
+  type: state.postgresql
+  metadata:
+  - name: connectionString
+    value: "<CONNECTION STRING>"
```

Comment:
Can you provide an example here? We have found that with an example, this helps people. Also **if** you can provide some guidance on how you get the connection string, that would help.

Reply:
I added an example and more guidance for how to create the connection string.